src/js/experiments/gettingStartedGuide.js

```diff
@@ -0,0 +1,33 @@
+// https://app.optimizely.com/v2/projects/16812830475/experiments/21268663100/variations
+
+/**
+ * Changes "Your First Green Build" in the docs sidebar to "Quickstart Guide"
+ * Adds a "NEW" badge to the sidebar item as well.
+ **/
+function addNewBadgeToSidebar() {
+  const NEW_SIDEBAR_HTML = `
+    <a class="" href="/docs/2.0/getting-started/"
+       data-section="getting-started" data-proofer-ignore="">
+      <span>Quickstart Guide</span>
+      <img style="margin: -2px 0px 0px 4px;" src="/docs/assets/img/compass/new.svg" alt="New">
```

Comment:
should we use a dynamic path for the img src for preview?

Reply:
We are unable to access {{site.baseurl}} in js. @rsylvian - do you think we should just use base64?

Reply:
Why do we need an image for that? can we just do it in CSS?